src/DebuggerForm.cpp

```diff
@@ -669,7 +691,8 @@ void DebuggerForm::createForm()
 	        stackView,  SLOT(setStackPointer(quint16)));
 	connect(disasmView, SIGNAL(breakpointToggled(int)),
 	                    SLOT(toggleBreakpoint(int)));
-
+	// connect(bpView, &BreakpointViewer::breakpointRequested, this,
+	//                     &DebuggerForm::toggleBreakpoint);
```

Comment:
Please don't keep commented-out code in the final version.
Unless there's a good reason for it, but then mention that in a comment.